src/content/structured/components/text-field/code.mdx

```diff
@@ -202,6 +202,41 @@ export const snippetsDisabled = [
   </IcTextField>
 </ComponentPreview>
 
+### Size small
+
+export const snippetsSmall = [
```

Comment:
This small functionality doesn't seem to work. In the page, the text field appears the same size as other text fields.